ballerina/os.bal

```diff
@@ -58,3 +58,58 @@ public isolated function getUserHome() returns string = @java:Method {
     name: "getUserHome",
     'class: "io.ballerina.stdlib.os.nativeimpl.GetUserHome"
 } external;
+
+# Sets the value of the environment variable named by the key. 
+# Note that the parameter key cannot be an empty string or "==" sign.
+# ```ballerina
+# os:Error? err = os:setEnv("BALCONFIGFILE", "/path/to/Config.toml");
+# ```
+#
+# + key - Key of the environment variable
+# + value - Value of the environment variable
+# + return - error if setting the environment variable fails, () otherwise
+public isolated function setEnv(string key, string value) returns Error? {
+    if key == "" {
+        return error Error("The parameter key cannot be an empty string");
+    } else if key == "=="  {
+        return error Error("The parameter key cannot be == sign");
+    } else {
+        return setEnvExtern(key, value, false);
+    }
+}
+
+# Removes a single environment variable from the system if it exists.
+# Note that the parameter key cannot be an empty string.
+# ```ballerina
+# os:Error? err = os:unsetEnv("BALCONFIGFILE");
+# ```
+#
+# + key - Key of the environment variable
+# + return - error if unsetting the environment variable fails, () otherwise
+public isolated function unsetEnv(string key) returns Error? {
+    if key == "" {
+        return error Error("The parameter key cannot be an empty string");
+    } else {
+        return setEnvExtern(key, "", true);
+    }
+}
+
+isolated function setEnvExtern(string key, string value, boolean remove) returns Error? = @java:Method {
+    name: "setEnv",
+    'class: "io.ballerina.stdlib.os.nativeimpl.SetEnv"
+} external;
+
+# Returns a map of environment variables.
+# ```ballerina
+# map<string> envs = os:listEnv();
+# ```
+#
+# + return - map of environment variables
```

Comment:
```suggestion
# + return - Map of environment variables
```

Reply:
Updated in [c82608f](https://github.com/ballerina-platform/module-ballerina-os/pull/423/commits/c82608fee36ec9c41f67334ffd424592d7a81d92)